### ISSUE TITLE
Have a dark theme available

### ISSUE THREAD

**Issue by GeorgiaHCA** [alan-turing-institute/AutSPACEs#603]
### Summary

AutSPACEs is not accessible to people with some visual sensory sensibilities. Having a dark theme would allow more users to feel comfortable with the website. 

### What needs to be done?

- [ ] Give users the option of changing the settings to include a dark theme - allow this to be easily adjusted and put it in a clear place so that it is easy to find 

### Who can help?

Please put suggestions of where and how to include the dark theme option in the comments below

* 
* 

---

**Comment by GeorgiaHCA** [comment]
Oops multiple similar issues
